deisctl/client/client.go

```diff
@@ -240,6 +241,20 @@ Usage:
 	return cmd.ListUnits(c.Backend)
 }
 
+func (c *Client) Machines(argv []string) error {
+	usage := `Enumerate the current hosts in the cluster
```

Comment:
This is an awesome PR, so this is nitpicking mode here.  Does "enumerate" actually imply some ordinality to the results that actually isn't present?  Might it be better to just say, "List the current hosts in the cluster?"